x-pack/test_serverless/shared/config.base.ts

```diff
@@ -159,7 +159,7 @@ export default async () => {
       try: 120 * 1000,
       waitFor: 20 * 1000,
       esRequestTimeout: 30 * 1000,
-      kibanaReportCompletion: 60 * 1000,
+      kibanaReportCompletion: 80 * 1000,
```

Comment:
I think with this change, the timeout override in the test file is no longer needed.

Reply:
I think so, I removed it

Reply:
Oh, right, I had to reload the browser page to see that change :see_no_evil: